apps/docs/src/app/core/component-docs/resizable-card-layout/examples/resizable-card-layout-example.component.ts

```diff
@@ -0,0 +1,281 @@
+import { Component, TemplateRef } from '@angular/core';
```

Comment:
here the example has 3 files but on the doc we see only the html one:
![Screen Shot 2021-05-17 at 6 03 07 AM](https://user-images.githubusercontent.com/4380815/118471367-af48b880-b6d5-11eb-9826-71efa8c8c4aa.png)
![Screen Shot 2021-05-17 at 6 03 45 AM](https://user-images.githubusercontent.com/4380815/118471368-afe14f00-b6d5-11eb-8e5a-fc097eb2567a.png)


Same for the other 2 examples

Reply:
fixed, now showing all 3 files.